cmd/static/index.html

```diff
@@ -40,7 +40,7 @@
 	<!-- head bar begin. -->
```

Comment:
keeping with my suggestion above, all of these files that used to live under `web/static` would now live under `/cmd/edgex-ui-server/static`